components/Head.js

```diff
@@ -1,23 +1,14 @@
 import React from "react";
+import Link from "next/link";
+
 import styles from "./Head.module.scss";
-import { useContext } from "react";
-import AppContext from "../AppContext";
 
 const Head = () => {
-    const value = useContext(AppContext);
-    let { userSelected } = value.state;
-    console.log("userSelected", userSelected);
     return (
         <div className={styles.container}>
-            {userSelected.name ? (
-                <div className={styles.textBox}>
-                    <span>Name:{userSelected.name}</span>
-                    <span>Email:{userSelected.email}</span>
-                    <span>Age:{userSelected.age}</span>
-                </div>
-            ) : (
-                <p>No data found</p>
-            )}
+            <Link className={styles.myAccount_link} href={"/myAccount"}>
+                my account
+            </Link>
```

Comment:
my Account link should be visible only when the user is logged in or in our case if the user data exists in context.